example_config/EUR-Lex/kim_cnn.yml

```diff
@@ -0,0 +1,33 @@
+# data
+data_dir: data/EUR-Lex
+data_name: EUR-Lex
+min_vocab_freq: 1
+max_seq_length: 500
+
+# train
+seed: 1337
+epochs: 50
+batch_size: 64
+optimizer: adam
+learning_rate: 0.0005099137446356937
+weight_decay: 0
+patience: 5
+
+# eval
+eval_batch_size: 256
+monitor_metrics: [P@1, P@3, P@5]
+val_metric: P@1
+
+# model
+model_name: KimCNN
+num_filter_map: 128 # filter channels
+#filter_size: 4 # no filter sizes
+filter_sizes: [2, 4, 8]
+strides: [1, 1, 1]
+dropout: 0.2
+
+# pretrained vocab / embeddings
+embed_file: glove.6B.300d
+
+# others
+data_workers: 4
```

Comment:
this is the default value. Do we now need this in all yml files?

Reply:
Solved.